usr/include/stdll.h

```diff
@@ -88,7 +88,8 @@ typedef CK_RV (CK_PTR ST_C_OpenSession) (STDLL_TokData_t *tokdata,
                                          CK_SESSION_HANDLE_PTR phSession);
 
 typedef CK_RV (CK_PTR ST_C_CloseSession) (STDLL_TokData_t *tokdata,
-                                          ST_SESSION_T *hSession);
+                                          ST_SESSION_T *hSession,
+                                          CK_BBOOL in_fork_initializer);
```

Comment:
i possibly misunderstood this patch but is the additional function param only used to propagate "in_child_fork_initializer" which is a global variable anyway ?

Reply:
Yes it is to propagate in_child_fork_initializer. 
Variable in_child_fork_initializer is a global variable in the API shared library, but it is not available in the token shared libraries, so it has to be passed over.